### PR TITLE
add short tutorial about namespaces (#280)

### PR DESCRIPTION
**Pull request by petermax2**
I don't know how useful this is as we already have ```doc/NAMESAPCES.md```. 

Please give me feedback about how short (or long) you want the tutorial to be.

### CODE REVIEW

File: doc/tutorials/namespaces.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Having namespaces enables both admins and users to set specific parts of the application's configuration.
+
+## How it works ##
+
+The idea is to call ```kdbGet()``` to retrieve the root key for the application.
+Looking for a specific part of the configuration is done by ```ksLookup()```.
+
+The documentation provides the following example to illustrate the intended usage:
+
+	if (kdbGet(handle, "user/myapp", myConfig, 0 ) == -1)
```
Comment (markus2330) [inline doc/tutorials/namespaces.md]: should be Markdown source

**Comment by markus2330** [comment]
It would be useful if you also give shell examples (more tutorial style, not throw words like "cascading keys" to the user without giving background)

File: doc/tutorials/namespaces.md (hunk 1 above)
Comment (petermax2) [inline doc/tutorials/namespaces.md]: Do you mean the "```" inline code elements? Everything else should be 100% Markdown conform.

**Comment by markus2330** [comment]
Thank you, nicely done!

**Pull request merged**